app/assets/stylesheets/application.scss

```diff
@@ -143,37 +175,33 @@ h6 {
 
 }
 
-.item-background-img {
-
-}
-
-#main {
+.main {
   position: relative;
   text-align: center;
+  margin-bottom: 35px
   h1 {
-    color: white;
+    color: #666666;
   }
 
 }
 
-#main:after {
+.main:after {
    content : "";
    display: block;
    position: absolute;
    top: 0;
    left: 0;
-   background-image: url("/assets/plate.jpg");
    width: 100%;
    height: 100%;
-   opacity : 0.4;
    z-index: -1;
+   margin-bottom: 35px
```

Comment:
Line should be indented 2 spaces, but was indented 3 spaces<br>Declaration should be terminated by a semicolon